src/app/shared/entry.ts

```diff
@@ -220,7 +220,15 @@ export abstract class Entry implements OnInit, OnDestroy {
       return null;
     }
     const selectedTag = this.selectVersion(versions, urlVersion, defaultVersion);
-    return selectedTag || versions.reduce((a, b) => (b.last_built > a.last_built ? b : a));
+    return (
+      selectedTag ||
+      versions.reduce((a, b) => {
+        if (!b.last_built && !a.last_built) {
+          return b['dbUpdateDate'] > a['dbUpdateDate'] ? b : a;
```

Comment:
Just curious, why are you using this syntax instead of `b.dbUpdateDate`? Does TypeScript still do type checking this way?

Reply:
was back before the webservice was ready